tests/cypress/integration/cmsWorkflow/unlockResubmit.spec.ts

```diff
@@ -129,9 +129,6 @@ describe('dashboard', () => {
                 cy.wait('@fetchSubmission2Query', { timeout: 50000 })
                 cy.findByTestId('updatedSubmissionBanner').should('exist')
 
-                cy.findByText('Back to state dashboard').click()
```

Comment:
was this flaking or what was happening here?

Reply:
This was something was testing before @macrael fixed the issue with signing out.